docs/advanced.md

```diff
@@ -26,3 +26,7 @@ bg_color = viewer.get_background_color()
 ```python
 print(bg_color)
 ```
+
+This has [not yet applied in
+JupyterLite](https://github.com/InsightSoftwareConsortium/itkwidgets/issues/730)
```

Comment:
"This has not yet applied in JupyterLite" -> "This has not yet been applied in JupyterLite" or "This is not yet applied in JupyterLite"?